---
int32 errcode
string errmsg
string[] meshNames
